File: BackEnd/backend/src/main/java/com/fitapp/backend/application/dto/exercise/ExerciseResponse.java
package com.fitapp.backend.application.dto.exercise;

import java.time.LocalDateTime;
import java.util.Map;
import lombok.Data;

@Data
public class ExerciseResponse {
    private Long id;
    private String name;
    private String description;
    private Long sportId;
    private Long userId;
    private Boolean predefined;
    private Map<String, String> parameterTemplates;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/response/exercise/ExerciseResponse.kt
package com.fitapp.appfit.response.exercise

data class ExerciseResponse(
    val id: Long,
    val name: String,
    val description: String?,
    val sportId: Long?,
    val sportName: String?,
    val isPredefined: Boolean,
    val parameterTemplates: Map<String, String>
)